### ISSUE TITLE
Union Cases on Top of Completion List

### ISSUE THREAD

**Issue by SchlenkR** [dotnet/fsharp#15241]
Given

```fsharp
type DU = A | B

let f (x: DU) = ()
```

results in a completion list that doesn't priorize the labels of type `DU`. I suggest to the labels on top of the list.

![grafik](https://github.com/dotnet/fsharp/assets/472222/949c8071-6828-48ad-a003-ca5970c28049)

**Links**

https://twitter.com/vzarytovskii/status/1660202383605891074

**Comment by edgarfgp** [comment]
Similar to https://github.com/dotnet/fsharp/issues/6996 ?

**Comment by T-Gro** [comment]
@edgarfgp Yes. I think also for this issue the best bet would be trying to revive https://github.com/dotnet/fsharp/pull/12990/files

**Comment by edgarfgp** [comment]
> @edgarfgp Yes. I think also for this issue, the best bet would be trying to revive https://github.com/dotnet/fsharp/pull/12990/files

@T-Gro It seems that Kerams was close to completing this one. But it needed to have the deeper review required to be merged. Maybe the F# team can pick this one :)
